config/locales/en.yml

```diff
@@ -0,0 +1,2 @@
+en:
```

Comment:
Since we aren't using this actually, why not delete this file and lines 13/14 of lib/bible_bot.rb?

Reply:
i18n raises:

```
I18n::InvalidLocale:
         :en is not a valid locale
```

without it. I was following the [gem's guide](I18n::InvalidLocale:
         :en is not a valid locale) for using it without Rails but I can look through more documentation to see if there is an alternative. 

There also may be a Ruby-only solution by using String's `encode` method but I couldn't get it to work.
https://ruby-doc.org/core-2.5.1/String.html#method-i-encode

Reply:
Nah, this is fine. I don't think character encoding is what you want.

Reply:
I didn't think so either. It looked like a lot more configuration vs the convention of `transliterate.`